events_tickets_limit/models/event_ticket.py

```diff
@@ -0,0 +1,11 @@
+from odoo import _, models, fields , api
```

Comment:
```suggestion
from odoo import models, fields
```

No need of '_' if we are not translating anything. `api` not required